src/CILogViewer.php

```diff
@@ -6,58 +6,43 @@
  */
 namespace CILogViewer;
 
-defined('BASEPATH') OR exit('No direct script access allowed');
-defined('APPPATH') OR exit('Not a Code Igniter Environment');
-
-
 class CILogViewer {
 
-    private $CI;
-
     private static $levelsIcon = [
+        'CRITICAL' => 'glyphicon glyphicon-error-sign',
         'INFO'  => 'glyphicon glyphicon-info-sign',
         'ERROR' => 'glyphicon glyphicon-warning-sign',
         'DEBUG' => 'glyphicon glyphicon-exclamation-sign',
         'ALL'   => 'glyphicon glyphicon-minus',
     ];
 
     private static $levelClasses = [
+        'CRITICAL' => 'danger',
```

Comment:
We need to add the following log level as well:
- NOTICE
- WARNING
- ALERT
- EMERGENCY

You can use the following code snippets:

```
private static $levelsIcon = [
        'CRITICAL' => 'glyphicon glyphicon-error-sign',
        'NOTICE' => 'glyphicon glyphicon-error-sign',
        'WARNING' => 'glyphicon glyphicon-warning-sign',
        'ALERT' => 'glyphicon glyphicon-error-sign',
        'INFO'  => 'glyphicon glyphicon-info-sign',
        'ERROR' => 'glyphicon glyphicon-warning-sign',
        'DEBUG' => 'glyphicon glyphicon-exclamation-sign',
        'EMERGENCY' => 'glyphicon glyphicon-error-sign',
        'ALL'   => 'glyphicon glyphicon-minus',
    ];

    private static $levelClasses = [
        'CRITICAL' => 'danger',
        'INFO'  => 'info',
        'ERROR' => 'danger',
        'DEBUG' => 'warning',
        'NOTICE' => 'notice',
        'WARNING' => 'warning',
        'EMERGENCY' => 'emergency',
        'ALERT' => 'alert',
        'ALL'   => 'muted',
    ];
```